### DISCUSSION TITLE
Windows Schedule Powershell Task

### DISCUSSION THREAD

**Discussion by djarbz** [bdd/runitor#171] [General]
I've been banging my head on this for quite some time and I'm getting nowhere.
I have my ST setup to run as SYSTEM and with highest priv.
Program is set to the full path of the runitor binary.
Args are set to:
`-api-url 'https://domain.tld/ping' -uuid 'UUID' -- powershell -NoProfile -NoLogo -NonInteractive -ExecutionPolicy Bypass -File RenewCertificate.ps1`
Start In is set to the directory containing the ps1 file.

The script runs, but I don't see anything in my HC dashboard.
If I run the command as administrator or as SYSTEM, it works just fine.
```powershell
PS C:\ASSETS\Scripts> C:\ASSETS\Tools\runitor-v1.2.0-build.2-windows-amd64.exe -api-url 'https://domain.tld/ping' -uuid 'UUID' -- powershell -NoProfile -NoLogo -NonInteractive -ExecutionPolicy Bypass -File RenewCertificate.ps1
```

**Comment by bdd** [comment]
I'm not familiar with the Windows environment. So hopefully some other user will see this discussion and chime in to share their experience.
